src/feed_handler.py

```diff
@@ -83,7 +92,26 @@ def _update_height(self, clob_pair_id: int, new_block_height: int):
                 f"Block height decreased from "
                 f"{self.heights[clob_pair_id]} to {new_block_height}"
             )
-    
+
+    def _handle_subaccounts(self, update: subaccounts.StreamSubaccountUpdate):
+        """
+        Handle the StreamSubaccountUpdate message, updating the local subaccount state.
+        """
+        parsed_subaccount = subaccounts.parse_subaccounts(update)
+        subaccount_id = parsed_subaccount.subaccount_id
+
+        if update.snapshot or subaccount_id not in self.subaccounts:
```

Comment:
For the snapshot can we follow the pattern [here](https://github.com/dydxprotocol/grpc-stream-client/blob/add120829fedc95383b26649a02bf08996d0760f/src/feed_handler.py#L170-L180) of ignoring subsequent snapshots/raising this error?

Reply:
done